cmd/image/qcow2ova/qcow2ova.go

```diff
@@ -159,10 +160,23 @@ Qcow2 images location:
 		if opt.ImageDist != "coreos" && opt.OSPassword == "" && !opt.OSPasswordSkip {
 			var err error
 			opt.OSPassword, err = GeneratePassword(12)
-			klog.Infof("Autogenerated OS root password is: %s", opt.OSPassword)
 			if err != nil {
 				return err
 			}
+			passwordfile, err := os.Create("password.txt")
+			if err != nil {
+				klog.Fatalf("failed to create password file, err: %v", err)
+				return err
+			}
+
+			passwordfile.WriteString(fmt.Sprintf("Root Password : %s", opt.OSPassword))
+			cwd, err := os.Getwd()
+			if err != nil {
+				return err
+			}
+
+			klog.Infof("Autogenerated OS root password is updated in file :  %s/%s", cwd, passwordfile.Name())
```

Comment:
Ref: https://pkg.go.dev/path/filepath#Join
```suggestion
			klog.Infof("Autogenerated OS root password is updated in %s", filepath.Join(cwd, passwordfile.Name()))
```

Reply:
Done.